implicits/src/com/rallyhealth/weepickle/v1/implicits/DefaultTos.scala

```diff
@@ -256,6 +256,9 @@ trait DefaultTos extends com.rallyhealth.weepickle.v1.core.Types with Generated
 
         def subVisitor = implicitly[To[T]]
       }
+      override def visitString(cs: CharSequence): Array[T] = {
+        Base64.getDecoder.decode(cs.toString).asInstanceOf[Array[T]]
```

Comment:
why?

Reply:
Doug assumed binary data serialized to Json should be encoded as b64 strings. So if you serialize binary data to Json and then attempt to deserialize back to the source structure, the reverse assumption needs to be made or deserialization fails.